EnglishFilter/sections/specific.txt

```diff
@@ -297,6 +297,7 @@ linkjust.com,updatesmarugujarat.in,mexashare.com,links4u.me,linku.us,skips.link,
 collive.com##a[href*="&ad_url="]
 collive.com##.premium-post-ad
 m.xcum.com##.mob_under_player
+roadtovr.com##footer > .td-post-content
```

Comment:
`roadtovr.com##footer > div.td-post-content`. I'd use the extended css, but this rule looks fine.